lumicks/pylake/scan.py

```diff
@@ -97,8 +97,7 @@ def _plot(self, image, frame=1, image_handle=None, **kwargs):
             image = image[frame - 1]
 
         axes = self._ordered_axes()
```

Comment:
Consider moving this in closer proximity to the plot command, since that is now the first use of these axes. Helps to keep variables more local to their use.